src/odemis/gui/util/widgets.py

```diff
@@ -180,8 +180,12 @@ def _on_value_change(self, evt):
         """ This method is called when the value of the control is changed.
         it moves the axis to the new value.
         """
+        evt.Skip()
+
         try:
             if self._future is not None and not self._future.done():
```

Comment:
In this case, yes I think it's possible, because Future are always considered "True". More and more I tend to leave this explicit `is not None` for every "complex" object, because sometimes some objects are designed to be considered `False`, and it's pretty hard to know which one. An example is wx objects, which are considered `False` when they have been `Destroy()'ed` (but they might still contain useful data on it). numpy arrays even raise an error when compared this way.